dashboard/app/views/plc/user_course_enrollments/show.html.haml

```diff
@@ -0,0 +1,17 @@
+%p#notice= notice
+
+%p Hello #{@user_course_enrollment.user.name}, here is your progress for #{@user_course_enrollment.plc_course.name}
+
+-@user_course_enrollment.module_assignments.each do |module_enrollment|
+  %p
+    %strong #{module_enrollment.plc_learning_module.name}
+    - module_enrollment.task_assignments.each do |task_assignment|
+      %li #{task_assignment.plc_task.name} - #{task_assignment.status}
+
+-if @user_course_enrollment.module_assignments.count == 0
+  %p
+    You have not taken the evaluation quiz. Once you have done so, we can enroll you in appropriate modules.
+    = link_to 'Take evaluation quiz', "/plc/enrollment_evaluations/#{@user_course_enrollment.id}/perform_evaluation"
```

Comment:
I think you can use `perform_evaluation_path(@user_course_enrollment)` here? It's nice to use the route helpers instead of generating urls in general so you don't have to search around for url references if you change the routes (and if you delete one you get an error instead of a silently broken link).